src/main/java/io/weaviate/client/v1/aliases/api/AliasAllGetter.java

```diff
@@ -0,0 +1,36 @@
+package io.weaviate.client.v1.aliases.api;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import io.weaviate.client.Config;
+import io.weaviate.client.base.BaseClient;
+import io.weaviate.client.base.ClientResult;
+import io.weaviate.client.base.Response;
+import io.weaviate.client.base.Result;
+import io.weaviate.client.base.http.HttpClient;
+import io.weaviate.client.v1.aliases.api.AliasAllGetter.ResponseBody;
+import io.weaviate.client.v1.aliases.model.Alias;
+
+public class AliasAllGetter extends BaseClient<ResponseBody> implements ClientResult<Map<String, Alias>> {
```

Comment:
here's also a possibility of pass a `class` query argument which filters all `aliases` to the ones pointing at a given `class`

Reply:
Thanks, I didn't notice that parameter 🤭